react.component/src/form/Confirmation/confirmation.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, test } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { Confirmation } from './Confirmation';
+
+describe('react form page', () => {
+  test('Confirm message renders properly', () => {
+    render(<Confirmation />);
+    const confirmation = screen.getByTestId('confirmation');
```

Comment:
`getByTestId` will throw an error in case of missing element, so you can remove the line down below with the check